src/super_gradients/training/transforms/keypoints/keypoints_rescale.py

```diff
@@ -0,0 +1,72 @@
+import random
+import cv2
+import numpy as np
+from typing import Tuple, List
+from super_gradients.common.registry import register_transform
+from super_gradients.common.object_names import Transforms, Processings
+from super_gradients.training.samples import PoseEstimationSample
+
+from .abstract_keypoints_transform import AbstractKeypointTransform
+
+
+@register_transform(Transforms.KeypointsRescale)
+class KeypointsRescale(AbstractKeypointTransform):
+    """
+    Resize image, mask and joints to target size without preserving aspect ratio.
+    """
+
+    def __init__(self, height: int, width: int, interpolation: int = cv2.INTER_LINEAR, prob: float = 1.0):
+        """
+
+        :param max_sizes: (rows, cols) - Maximum size of the image after resizing
+        :param interpolation: Used interpolation method for image
+        :param prob: Probability of applying this transform
+        """
+        super().__init__()
+        self.height = height
+        self.width = width
+        self.interpolation = interpolation
+        self.prob = prob
+
+    def apply_to_sample(self, sample: PoseEstimationSample) -> PoseEstimationSample:
+        if random.random() < self.prob:
+            height, width = sample.image.shape[:2]
+            sy = self.height / height
+            sx = self.width / width
+
+            sample.image = self.apply_to_image(sample.image, dsize=(self.width, self.height), interpolation=self.interpolation)
+            sample.mask = self.apply_to_image(sample.mask, dsize=(self.width, self.height), interpolation=cv2.INTER_NEAREST)
+
+            sample.joints = self.apply_to_keypoints(sample.joints, sx, sy)
+            if sample.bboxes is not None:
+                sample.bboxes = self.apply_to_bboxes(sample.bboxes, sx, sy)
+
+            if sample.areas is not None:
+                sample.areas = np.multiply(sample.areas, sx * sy, dtype=np.float32)
+
+        return sample
+
+    @classmethod
+    def apply_to_image(cls, img, dsize: Tuple[int, int], interpolation):
+        img = cv2.resize(img, dsize=dsize, interpolation=interpolation)
+        return img
+
+    @classmethod
+    def apply_to_keypoints(cls, keypoints, sx, sy):
+        keypoints = keypoints.astype(np.float32, copy=True)
+        keypoints[:, :, 0] *= sx
+        keypoints[:, :, 1] *= sy
+        return keypoints
+
+    @classmethod
+    def apply_to_bboxes(cls, bboxes, sx, sy):
+        bboxes = bboxes.astype(np.float32, copy=True)
+        bboxes[:, 0::2] *= sx
+        bboxes[:, 1::2] *= sy
+        return bboxes
```

Comment:
Typing + docstring.
(Also, same question as above, was there a motivation for using `@ classmethod` instead of `@staticmethod` ?)